examples/03-Maxwell/Maxwell2D_DCConduction.py

```diff
@@ -178,6 +195,43 @@
 animated.azimuth_angle = 0
 animated.animate()
 
+################################################################################
+# Export model picture
+# ~~~~~~~~~~~~~~~~~~~~
+# Export model picture.
+
+model_picture = m2d.post.export_model_picture()
+
+################################################################################
+# Generate pdf report
+# ~~~~~~~~~~~~~~~~~~~
+# Generate a pdf report with output of simulation.
```

Comment:
```suggestion
# Generate PDF report
# ~~~~~~~~~~~~~~~~~~~
# Generate a PDF report with output of simulation.
```
Rule is to place file extension letters in all capital letters (and also not to use something like *.pdf)